mod/gc_onboard/views/default/welcome-steps/wire_modal.php

```diff
@@ -32,12 +32,24 @@
     }
 </style>
 <div class="row clearfix wire-button-holder">
-    <a aria-hidden="true" role="button" id="wirePopup" class="overlay-lnk btn btn-default gcconnex-edit-profile pull-right" href="#showWire" aria-controls="mid-screen"><?php echo elgg_echo('onboard:wireButton'); ?></a>
+    <a role="button" id="wirePopup" class="overlay-lnk btn btn-default gcconnex-edit-profile pull-right" href="#showWire" aria-controls="mid-screen"><?php echo elgg_echo('onboard:wireButton'); ?></a>
 
+	                <?php if(!elgg_get_logged_in_user_entity()->hasVisitedWire){?>
+                      
+                <script>
+                    $(document).ready(function () {
+                        $('#wirePopup').delay(800).click();
+						$('#wirePopup').on('click', function(){
+							$('#showWire').focus();	
+						});
+                    });
+                </script>
+
+                      <?php }?>
 </div>
 
 
-<section class="wb-overlay modal-content overlay-def wb-popup-mid" id="showWire">
+<div class="wb-overlay modal-content overlay-def wb-popup-mid" id="showWire" tabindex="-1" aria-live="assertive">
```

Comment:
I'm not too familiar with Elgg, but I'm not seeing where dynamic content is coming into this section that would need a live region at all. Also, if this adds multiple live regions to the page, you're going to need code to syncronize them